pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "project_name"
+name = "pecha-org-categorizer"
```

Comment:
set name = "pecha_org_categorizer"

Reply:
I have updated the name of the whole project to pecha_org_tools